### ISSUE TITLE
PiedPiPer extension no longer working for Netflix

### ISSUE THREAD

**Issue by 9avin** [JoeKuhns/PiedPiPer.safariextension#30]
hi looks like netflix has broken the PIP function again in safari. i too use this all the time great work in creating this extension. cheers

**Comment by ToniWonKanobi** [comment]
Yeah, it's totes broken [again](https://github.com/JoeKuhns/PiedPiPer.safariextension/issues/5) 😢

**Comment by jareddyck** [comment]
stopped working for me too. I also recently updated to safari 11 - perhaps the update broke it?

**Comment by ToniWonKanobi** [comment]
@jareboomafoo Could be 😞
